crates/egui/src/input_state/mod.rs

```diff
@@ -1028,15 +1029,6 @@ impl PointerState {
 
         self.pos_history.flush(time);
 
-        self.velocity = if self.pos_history.len() >= 3 && self.pos_history.duration() > 0.01 {
-            self.pos_history.velocity().unwrap_or_default()
-        } else {
-            Vec2::default()
-        };
-        if self.velocity != Vec2::ZERO {
-            self.last_move_time = time;
-        }
-
         self.direction = self.pos_history.velocity().unwrap_or_default().normalized();
```

Comment:
The root cause is that last_move_time is not updated in certain situations (slow situations).